README.md

```diff
@@ -1,52 +1,167 @@
 # Ajax and Callbacks
-
 ## Objectives
-+ Explain what an API is and why it's used
-+ Explain what Ajax is and why it's used
-+ Make a `get` request using Ajax to append text to a page
-+ Explain what a callback is and why Ajax runs asynchronously
+- Explain what an API is and why it's used
+- Explain what Ajax is and why it's used
+- Make a get request using Ajax to append text to a page
+- Explain what a callback is and why Ajax runs asynchronously
+
+## Intro (half done)
+
+So far, you've been using a gem to access a web API.  Gems are responsible for handling all of the nitty-gritty.  They allow us to easily access/modify the data that's important to us. But what happens if the API we want to use doesn't have an associated gem?  We will need to handle everything ourselves.  If that's the case, we better learn more about what an API really is!
+
+## API Basics (done)
+
+We interact with web APIs through a set of urls. Each url defines a resource that we access.  The benefit of using a gem is that we don't have to build these urls ourselves.  The gems wrap the necessary code into neat little packages.  When we ask a gem for data, a gem will make a request to the API, handle the response and parse the data into a format we can use. Instead of relying on the magic of gems let's get our hands dirty.
+
+### Postman and the GitHub API (done)
+Let's see what's actually happening when a gem makes a request to an API. Ok, but how do we do this without a gem? We could write some ruby code, but let's try using a tool called Postman. Postman is an easy to use Chrome extension that lets us make different web requests.  It easily allows us to interact with web APIs. For our exercise, we are going to work with the GitHub API to retrieve information about the Ruby on Rails GitHub repository. To get started we need to setup Postman.
+
+#### Postman Installation (almost done)
+
+1. Visit https://www.getpostman.com and install the Chrome extension.
+2. Once you have it installed, open Postman from the Chrome Apps menu.
+3. Skip the signup and go straight to the app. 
+
+Great!  Now you're all set up to make your own API requests.  As you know, Rails is a large open source project with thousands of contributors.  Those contributors make a hefty amount of commits.  The GitHub API allows us to retrieve the list of commits made to the Rails repository.
```

Comment:
This section is great at easing the student into parameterizing their calls to the API — thinking about gems in the background makes the analogy clear.